### ISSUE TITLE
Mac Client keeps prompting for server address and credentials

### ISSUE THREAD

**Issue by RubenGarcia2081** [owncloud/client#7155]
Hello everyone,
having an issue with ownlcoud client on Mac  OSX

already tried reinstalling client and clearing out keychain entries

### Expected behaviour
When quitting owncloud and starting again, the client should log in automatically

### Actual behaviour
After successful login with the client and saving of credentials owncloud client promts for server and user/password again when restarted

### Steps to reproduce
1. Install client
2. Enter server url and credentials
3. Tick saving credentials (confirm in keychain)
4. complete sync configuration and sync
5. close/terminate owncloud
6. restart owncloud

### Server configuration
Operating system:

Web server: Hostingprovider iway.ch

Database:

PHP version:

ownCloud version: 10.0.10 (stable)

Storage backend (external storage):

### Client configuration
Client version: 2.5.4 (build 11456)

Operating system: Mac OSX Mojave 10.14.4

OS language: German

Qt version used by client package (Linux only, see also Settings dialog): 5.12.1

Client package (From ownCloud or distro) (Linux only): owncloud

Installation path of client: Applications folder

### Logs

The keychainentry for owncloud is saved (user/password and server details) and owncloud client has access to it.
Don't think any log would help as when details are entered everything works fine untill client is closed or computer restarted.

Thank you for your support

**Comment by guruz** [comment]
Please open the "Keychain Access" utility on your Mac and double check if credentials really got saved or if you have a bogus setting there
https://support.apple.com/de-de/guide/keychain-access/what-is-keychain-access-kyca1083/mac

**Comment by RubenGarcia2081** [comment]
> Please open the "Keychain Access" utility on your Mac and double check if credentials really got saved or if you have a bogus setting there
> https://support.apple.com/de-de/guide/keychain-access/what-is-keychain-access-kyca1083/mac

An entry in the Keychain Access is created. I deleted the entry previous to the fresh install and a new entry was created once I loged in again with the client.
What format should the proper entry have - maybe it saves it wrong?

**Comment by guruz** [comment]
I don't know. It works for me and seeminlgly for everyone else.
:-/

**Comment by RubenGarcia2081** [comment]
My entry looks as followed:
Name: ownCloud
Kind: application password
Account: user@domain:serverurl:0
Where: ownCloud

and as mentioned ownCloud is listed under "Access Control" tab

could you compare the account entry format with yours?

**Comment by guruz** [comment]
Yes looks like that, additionally in other tab:


<img width="544" alt="Screen Shot 2019-04-18 at 14 42 55" src="https://user-images.githubusercontent.com/959327/56361694-4e91dd80-61e8-11e9-8bcc-55a70d0f50f5.png">


If you don't think you have a wrong setting on your side, maybe you need to run the client with the log window and check for possible hints (F12 key) or https://central.owncloud.org/t/how-to-enable-logging-for-desktop-client/854

**Comment by RubenGarcia2081** [comment]
under Attributes does your Account information als resemble this format: "user@domain:serverurl:0" ?

**Comment by ckamm** [comment]
@RubenGarcia2081 While I can't verify directly, that is indeed the expected string for the entry. 

Sometimes the logs can give us more information about what happens - though keychain access is handled through a library and sometimes doesn't have as much error information as we'd like. Look at the startup log and check for "credential" or "keychain" entries. Alternatively you could send me the log at mail at ckamm de.

**Comment by RubenGarcia2081** [comment]
So I ran the log and it looks like ownCloud can't save the configuration and therefore will always prompt to the wizard on startup.
As mentioned before an entry in the keychain is created and after reviewing the logs I don't think my issue lies there but rather that ownCloud wont even look for it due to the aforementioned issue.

I added a censored version of the log as I'm not certain why the configuration can't be saved. 
As far as I can see the relevant lines are the first 3 as the result of the problem and lines 29, 33 and 37 for the cause (below are these 6 lines):

Line 01 `05-02 09:39:04:308 [ info gui.folder.manager ]:	Setup folders from  "/Users/user/Library/Preferences/ownCloud/folders" (migration)`
Line 02 `05-02 09:39:04:325 [ info sync.clientproxy ]:	Set proxy configuration to use system configuration`
Line03 `05-02 09:39:04:626 [ info gui.application ]:	No configured folders yet, starting setup wizard`

Line29 `05-02 09:39:55:281 [ info gui.account.manager ]:	Saved account settings, status: QSettings::AccessError`
Line33 `05-02 09:39:55:282 [ info gui.account.manager ]:	Saved all account settings, status: QSettings::AccessError`
Line37 `05-02 09:39:55:305 [ info gui.folder ]:	Saved folder "1" to settings, status QSettings::AccessError`

Once logged in ownCloud works as expected but like I said it wont persist a restart of the client.

I also had a look at the path in from Line01 and the only thing I could find in the Preferences folder is a read only file named ownCloud with an extended attribute. Unlike all the other files in that folder the ownCloud file belongs to root and group 'wheel'
MacOS is not my forte so I'm grateful for any support

[ownCloud_log.txt](https://github.com/owncloud/client/files/3137288/ownCloud_log.txt)
